### DISCUSSION TITLE
Student developer pack benefits, still not received after being approved.

### DISCUSSION THREAD

**Discussion by kfofkf123** [community/community#113162] [GitHub Education]
### Hi everyone,

Hi everyone! I have recently been approved at the student developer pack on the 15th March, its already been 3 days and I haven't received any updates or emails, Help please! I was once rejected because I forgot to upload the billing information and verify the 2FA but I reapplied and got approved.
![approval](https://github.com/community/community/assets/163091194/4c7f8e7a-9cb9-47bd-83bb-dadc9f778019)
![rejected](https://github.com/community/community/assets/163091194/92100400-f34c-439a-85b3-fad9d55830b0)

**Comment by samus-aran** [comment]
Hi @kfofkf123 👋 

When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this Approval status. However, during busy periods like the start of a new semester, it may take a bit longer. Check out our [Student Developer Pack Application FAQs & Common Rejection Reasons 🎒](https://github.com/orgs/community/discussions/111352#top) 👍 

- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

I'll be closing this discussion as you are within the 72 hours verification period. If you still haven't received your Global Campus benefits after 5 working days please feel free to re-open.
